### PR TITLE
Bug fix. Maakuntien poistaminen korjattu.

### PR DESCRIPTION
**Pull request by rapitkan**
The changes are mostly made by the formatter (Prettier). The actual relevant change is in changes-to-save.js lines 307 - 326.

### CODE REVIEW

File: src/components/02-organisms/CategorizedListRoot/Stage/index.js [hunk 1]
```diff
@@ -21,8 +21,10 @@ const Stage = props => {
       if (interval) {
         if (props.loopChanges[nodeIndex + 1]) {
           setNodeIndex(nodeIndex + 1);
-        } else {
+        } else if (props.isLoopEnabled) {
           setNodeIndex(0);
+        } else {
+          console.info("Procedure ended.");
```
Comment (jpy-valamis) [inline src/components/02-organisms/CategorizedListRoot/Stage/index.js]: What useful info does this carry in the overall log stream? At least add which procedure and some context, or remove altogether.

File: src/components/02-organisms/CategorizedListRoot/utils.js [hunk 2]
```diff
@@ -388,7 +381,7 @@ export const handleNodeMain = (
         );
       }, changesWithoutRootAnchor)
     : changesWithoutRootAnchor;
-
+  console.info(updatedChangesArr);
```
Comment (jpy-valamis) [inline src/components/02-organisms/CategorizedListRoot/utils.js]: Looks like debug logging into info

Comment (rapitkan) [reply src/components/02-organisms/CategorizedListRoot/utils.js]: I'll remove it.

File: src/components/02-organisms/CategorizedListRoot/Stage/index.js (hunk 1 above)
Comment (rapitkan) [reply src/components/02-organisms/CategorizedListRoot/Stage/index.js]: Stage component is used only with Storybook.

**Pull request merged**